src/apyfixedarray_wrapper.cc

```diff
@@ -368,7 +357,50 @@ void bind_fixed_array(nb::module_& m)
             :class:`bool`
 
             )pbdoc")
-
+        .def(
+            "swapaxes",
+            &APyFixedArray::swapaxes,
+            nb::arg("axis1"),
+            nb::arg("axis2"),
+            R"pbdoc(
+          Interchange two axes of an array.
+          Parameters
+          ----------
+          axis1 : int
+              First axis.
+          axis2 : int
+              Second axis.
+
+          Examples
+          --------
+          >>> from apytypes import APyFixedArray
+          >>> x = APyFixedArray.from_float([[1,2,3]], bits=5,frac_bits=0)
```

Comment:
```suggestion
          >>> x = APyFixedArray.from_float([[1, 2, 3]], bits=5, frac_bits=0)
```

And similarly elsewhere. Compare with the .pyi-file where the Python formatter has done its thing.